app/forms/two_factor_options_form.rb

```diff
@@ -60,7 +60,13 @@ def has_no_configured_mfa?
     mfa_user.enabled_mfa_methods_count == 0
   end
 
+  def platform_auth_only_option?
+    mfa_user.enabled_mfa_methods_count == 1 &&
+      mfa_user.webauthn_platform_configurations.count == 1
+  end
+
   def has_no_mfa_or_in_required_flow?
-    has_no_configured_mfa? || in_phishing_resistant_or_piv_cac_required_flow?
+    has_no_configured_mfa? || in_phishing_resistant_or_piv_cac_required_flow? ||
```

Comment:
Unrelated and kinda an edge case, but I think there's a bug in this existing logic where a user is forced to select an MFA method during a phishing-resistant-required or PIV-only authentication setup, even if they've already set up a phishing-resistant or PIV method. Also, we still show the "Skip for now" link in what we're trying to prevent here. We could probably improve the consistency of how we enforce requirements of second MFA + show/hide the "Skip for now" link.

I'll create a ticket.

Reply:
Here's the ticket: https://cm-jira.usa.gov/browse/LG-10487